Dockerfile

```diff
@@ -11,6 +11,7 @@ COPY --chown=node:node package.json yarn.lock .yarnrc.yml tsconfig*.json ./
 RUN --mount=type=cache,target=/root/.yarn yarn
 COPY --chown=node:node assets ./assets
 COPY --chown=node:node migrations ./migrations
+COPY --chown=node:node scripts/generate-abis.ts ./scripts/generate-abis.ts
```

Comment:
Should be:

```
COPY --chown=node:node scripts/generate-abis.js ./scripts/generate-abis.js
```

This step also needs to happen before `RUN --mount=type=cache,target=/root/.yarn yarn` as the `postinstall` script runs then.

Reply:
I corrected the name in bb1e9cb458ccabc93c988e0e1cfa537a0a08c0a7 and moved the step in f3d7016936fe0dc19b6d44754109af3581ed74a8.